src/main/resources/io/jenkins/plugins/ml/Server/config.jelly

```diff
@@ -0,0 +1,46 @@
+<!--
+  ~ The MIT License
+
+  ~ Copyright 2020 Loghi Perinpanayagam.
+
+  ~ Permission is hereby granted, free of charge, to any person obtaining a copy
+  ~ of this software and associated documentation files (the "Software"), to deal
+  ~ in the Software without restriction, including without limitation the rights
+  ~ to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+  ~ copies of the Software, and to permit persons to whom the Software is
+  ~ furnished to do so, subject to the following conditions:
+
+  ~ The above copyright notice and this permission notice shall be included in
+  ~ all copies or substantial portions of the Software.
+
+  ~ THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+  ~ IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+  ~ FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+  ~ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+  ~ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  ~ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+  ~ THE SOFTWARE.
+-->
+
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form">
+
+    <f:entry title="Name" field="serverName">
+        <f:textbox />
+    </f:entry>
+    <f:entry title="Server address" field="serverAddress">
+        <f:textbox  />
+    </f:entry>
+    <f:entry title="Launch timeout" field="launchTimeout">
+        <f:textbox  />
+    </f:entry>
```

Comment:
@Loghijiaha for either this or a new PR, would be good to have help files here too, that help users that may have questions such as:

- The timeout is in ms? s? mins?
- What's the format of the server address? Does it need a protocol? Or is it just a socket address?
- What does max results mean? I assume it's the number of results returned by the server. But it could be the maximum size in bytes too?

You've added helper files already to the job configuration item, so it might be easy to add it here too.

Reply:
Helper are very helpful. Definitely I add that